content/04.experiment-types.md

```diff
@@ -16,4 +16,80 @@ Proteomics experiments can be both qualitative and quantitative.
+3. Determine the cross-linking efficiency through binding energy and calculations.
+4. Determine the number is cross-links which then will help you know the molecular weight.
+
+There are still some challenges in XL-MS such as:
+1. minimum amount of cross-linked peptides in complex mixtures [@doi:10.1021/acs.analchem.7b04431]
+2. diversity of cross linked products [@doi:10.1021/acs.analchem.7b04431]
+
+However, even though these challenges are an issue of now, we can expect further advancement to where this should not be an issue.
+
+2. Hydrogen deuterium exchange mass spectrometry (HDX-MS)
+HDX-MS works by detecting changes in the mass of the ion in which the amide hydrogens of the protein backbone is exchanged with its surrounding solvents. [@doi:10.1038/s41592-019-0459-y]. The rate of exchange depends on the protein folding state, dynamics, and the properties of the amino acid sequence. [@doi:10.1038/s41592-019-0459-y; @doi:10.1039/C0CS00113A; @doi:10.1016/S0065-3233(08)60129-1; @doi:10.1017/S0033583500005217] Although using D2O as a solvent to make deuterium labeled sample is simple, HDX-MS requires a number of controls to ensure that experimental conditions are done in an unbiased manner. [@doi:10.1038/s41592-019-0459-y; @doi:10.1586/epr.10.109; @doi:/10.1007/s13361-012-0377-z; @doi:10.4155/bio.15.46] This will produce precise measurements and high reproducibility if done correctly.
+[@doi:10.1038/s41592-019-0459-y] gives recommendations on how to prep samples, conduct data analysis, and present findings in a detailed stepwise manner that will be useful to have if you are new to using this technique.
+
+3. Fast photochemical oxidation of proteins (FPOP ) [@doi:10.1074/jbc.REV119.006218]
+Fast photochemical oxidation of proteins (FPOP) is a laser based HRPF (hydroxyl radical protein footprinting) MS method that relies on the irreversible labeling of solvent-exposed amino acid side chains by hydroxyl radicals in order to understand structure elucidation of proteins. The laser is set at 248 nm and will cause the hydrogen peroxides to become hydroxyl radicals. [@doi:10.1016/j.jasms.2005.09.008; @doi:10.1016/j.ijms.2006.08.018] The flow rate and laser frequency are adjusted in the flow system so that each protein molecule is radiated only once. After they are radiated, the sample is collected to a tube that has catalase and free methionine in buffer so that the H2O2 and OH can be quench and prevent further unwarranted oxidation. Control samples are made through running the sample through the flow system without any radiation. Another experimental control is done by adding a radical scavenger (glutamine is popularly used) with the purpose of preventing over oxidation. [@doi:10.1002/pro.3408; @doi:10.1021/ac901054w] FPOP has wide application for proteins, to include being able to report fast protein folding and protein transient dynamics.
```

Comment:
```suggestion
FPOP is a laser based hydroxyl radical protein footprinting MS method that relies on the irreversible labeling of solvent-exposed amino acid side chains by hydroxyl radicals in order to understand structure elucidation of proteins.
A laser produces 248 nm light to cause hydrogen peroxide to become hydroxyl radicals [@doi:10.1016/j.jasms.2005.09.008; @doi:10.1016/j.ijms.2006.08.018]. 
The flow rate of solution through the capillary and laser frequency are adjusted so that each protein molecule is radiated only once. 
After they are radiated, the sample is collected to a tube that has catalase and free methionine in the buffer so that the H2O2 and hydroxyl radical are quenched to prevent further unwarranted oxidation, which may then modify residues that become exposed due unfolding after the initial labeling. 
Control samples are made through running the sample through the flow system without any radiation. 
Another experimental control is done by adding a radical scavenger (glutamine is popularly used) with the purpose of preventing over oxidation [@doi:10.1002/pro.3408; @doi:10.1021/ac901054w].
FPOP has wide application for proteins, to include being able to report fast protein folding and protein transient dynamics.
```